tests/benchmark/test_osu.py

```diff
@@ -6,6 +6,57 @@
 from .. import FakeCluster
 
 
+class TestOSUOptionsAttributes(unittest.TestCase):
+    def test_categories_assignment(self):
+        benchmark = OSU()
+        # completely replace attributes
+        benchmark.attributes = dict(
+            executable='/path/to/fake',
+            srun_nodes=['node01', 'node03', 'node05'],
+            categories=['osu_bw', 'osu_latency'],
+            options={
+                'osu_bw': '[-x, 600, -i, 400]',
+                'osu_latency': '[-x, 500, -i, 200]',
+            },
+        )
+
+        self.assertEqual(
+            benchmark.arguments,
+            {'osu_bw': '[-x, 600, -i, 400]', 'osu_latency': '[-x, 500, -i, 200]'},
+        )
+
+    def test_partial_override(self):
+        benchmark = OSU()
+        # completely replace attributes
+        benchmark.attributes = dict(
+            executable='/path/to/fake',
+            srun_nodes=['node01', 'node03', 'node05'],
+            categories=['osu_bw', 'osu_latency'],
+            options={'osu_bw': '[-x, 600, -i, 400]'},
+        )
+
+        for cat in benchmark.arguments:
+            if cat == 'osu_bw':
+                self.assertEqual(benchmark.arguments[cat], '[-x, 600, -i, 400]')
+            else:
+                self.assertEqual(benchmark.arguments[cat], OSU.DEFAULT_ARGUMENTS[cat])
+
+    def test_all_categories_assign(self):
+        benchmark = OSU()
+        # completely replace attributes
```

Comment:
Same remarks than above concerning the comment and irrelevant attributes assigned to `benchmark.attributes`. Besides can you update the comment so that it says what the test is doing?

Reply:
Done!